### DISCUSSION TITLE
Ability for app to run in the background?

### DISCUSSION THREAD

**Discussion by Lolcupcakes** [jordenyt/stable_diffusion_sketch#16] [Ideas]
I usually go for quite high resolutions and lots of steps to produce a high quality image, but it takes a decent amount of time to generate. Is it possible to allow the app to run while I'm doing something else? Either way, thank you for developing this app, I love it!

**Comment by jordenyt** [comment]
In the latest version, I have moved the process to foreground service. The app should be able to run in background when you are calling APi for image result.

**Reply by Lolcupcakes** [reply]
Ah, thank you! I appreciate the dedication to updating this app!
